workflow/resources/datavzrd/go-enrichment-template.yaml

```
@@ -133,3 +133,11 @@ views:
               }  
             spec-path: ?input.vega_waterfall
             vega-controls: true
+  go_terms_scatter:
+    dataset: go_enrichment
+    desc: ?f"The y-axis represents the False Discovery Rate (FDR), while the x-axis shows the effect of the GO term for the {wildcards.model} model. 
```

I think we should mention the tool here, and what exactly the x-axis value represents. Do you know, what value `goatools` actually puts into the `study_items:<value>` field? I just put `??XYZ??` where I didn't know what to say, and looking into goatools code and docs didn't help me find out... Maybe you already dug into this?
```suggestion
    desc: ?f"The y-axis represents the False Discovery Rate (FDR), while the x-axis shows the effect of the GO term for the {wildcards.model} model as determined by goatools.
    The effect is determined as the sum of all the absolute ??XYZ?? values across all genes in a GO term.
```